test/WebSites/InlineConstraintsWebSite/Constraints/ItemTypeConstraint.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.AspNet.Http;
+using Microsoft.AspNet.Routing;
+
+namespace InlineConstraintsWebSite.Constraints
+{
+    public class ItemTypeConstraint : IRouteConstraint
```

Comment:
Can we think of something else interesting to do here? (Maybe look on stackoverflow for an example). This constraint is trivial so it's not super useful as a scenario.